iterator/linked-list.py

```diff
@@ -0,0 +1,136 @@
+class Node:
```

Comment:
Your Pull Request title: ITERATOR-1021 Added linked-list.py

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']